app/livechat/client/views/app/livechatDepartmentForm.js

```diff
@@ -171,26 +178,31 @@ Template.livechatDepartmentForm.events({
 	'click .remove-agent'(e, instance) {
 		e.preventDefault();
 
-		let selectedAgents = instance.selectedAgents.get();
-		selectedAgents = _.reject(selectedAgents, (agent) => agent._id === this._id);
-		instance.selectedAgents.set(selectedAgents);
+		let departmentAgents = instance.departmentAgents.get();
+		departmentAgents = _.reject(departmentAgents, (agent) => agent._id === this._id);
+		instance.departmentAgents.set(departmentAgents);
 	},
 });
 
 Template.livechatDepartmentForm.onCreated(async function() {
 	this.department = new ReactiveVar({ enabled: true });
+	this.departmentAgents = new ReactiveVar([]);
 	this.selectedAgents = new ReactiveVar([]);
-	this.agents = new ReactiveVar([]);
 
-	const { users } = await APIClient.v1.get('livechat/users/agent');
-	this.agents.set(users);
+	this.onSelectAgents = ({ item: agent }) => {
```

Comment:
I'd recommend avoid using `curValue` whenever is possible.. it's an internal property, not even documented and so might change without further notice.

if `.get()` cannot be used because it will add a reactivity/computation, you can use `Tracker.nonreactive`:

```suggestion
	this.onSelectAgents = Tracker.nonreactive(({ item: agent }) => {
		const selectedAgents = this.selectedAgents.get();
		selectedAgents.push(agent);
		this.selectedAgents.set(selectedAgents);
	});